### PR TITLE
web: Add env vars for setting 1080p and 4k bitrates.

### PR DESCRIPTION
**Pull request by jallamsetty1**

### CODE REVIEW

File: web/rootfs/defaults/settings-config.js [hunk 1]
```diff
@@ -408,48 +408,108 @@ config.videoQuality = {};
...
-config.videoQuality.h264 = {
-    maxBitratesVideo: {
-        low: {{ .Env.VIDEOQUALITY_BITRATE_H264_LOW }},
-        standard: {{ .Env.VIDEOQUALITY_BITRATE_H264_STANDARD }},
-        high: {{ .Env.VIDEOQUALITY_BITRATE_H264_HIGH }},
-        ssHigh: {{ .Env.VIDEOQUALITY_BITRATE_H264_SS_HIGH }}
-    }
-}
+{{ if .Env.VIDEOQUALITY_BITRATE_AV1_STANDARD }}
+config.videoQuality.av1.low = {{ .Env.VIDEOQUALITY_BITRATE_AV1_STANDARD }};
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:418]: Low?

File: web/rootfs/defaults/settings-config.js [hunk 2]
```diff
@@ -408,48 +408,108 @@ config.videoQuality = {};
...
-config.videoQuality.vp8 = {
-    maxBitratesVideo: {
-        low: {{ .Env.VIDEOQUALITY_BITRATE_VP8_LOW }},
-        standard: {{ .Env.VIDEOQUALITY_BITRATE_VP8_STANDARD }},
-        high: {{ .Env.VIDEOQUALITY_BITRATE_VP8_HIGH }},
-        ssHigh: {{ .Env.VIDEOQUALITY_BITRATE_VP8_SS_HIGH }}
-    }
-}
+{{ if .Env.VIDEOQUALITY_BITRATE_AV1_HIGH }}
+config.videoQuality.av1.low = {{ .Env.VIDEOQUALITY_BITRATE_AV1_HIGH }};
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:422]: Low?

File: web/rootfs/defaults/settings-config.js [hunk 3]
```diff
@@ -408,48 +408,108 @@ config.videoQuality = {};
...
+{{ end -}}
+
+{{ if .Env.VIDEOQUALITY_BITRATE_AV1_SS_HIGH }}
+config.videoQuality.av1.low = {{ .Env.VIDEOQUALITY_BITRATE_AV1_SS_HIGH }};
+{{ end -}}
+
+config.videoQuality.h264 = {};
+
+{{ if .Env.VIDEOQUALITY_BITRATE_H264_LOW }}
+config.videoQuality.av1.low = {{ .Env.VIDEOQUALITY_BITRATE_H264_LOW }};
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:440]: H264

**Review by saghul — CHANGES_REQUESTED** [review]
Lots of c&p errors

**Comment by jallamsetty1** [comment]
> Lots of c&p errors

Sorry about it! Fixed them.

**Review by saghul — APPROVED** [review]
🙌

**Pull request merged**